mava/wrappers/jaxmarl.py

```diff
@@ -157,16 +157,23 @@ def __init__(self, env: MultiAgentEnv, timelimit: int = 500):
 
         self.agents = list(self._env.observation_spaces.keys())
 
+        # Action mask fn
+        if hasattr(self._env, "get_avail_actions"):
+            self.get_avail_actions = self._get_avail_actions
+        else:
+            self.get_avail_actions = lambda _: jnp.ones(self._action_shape)
```

Comment:
If we can't get rid of the `hasattr` this is my suggestion
```suggestion
        # True if the wrapped env has a function to get the action mask
        has_action_mask = hasattr(self._env, "get_avail_actions")
```